### PR TITLE
codingminds_cappstone

### PR DESCRIPTION
**Pull request by FrankMugagga**
I created tow pages;About and home that are responsive for a summit: Coding Minds International Summit 2015
When you click on the logo, it redirects you to the home page
The project is also with a mobile menu that can redirect to both the home page and about page
There is a section called featured speakers that loads dynamically

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+                    <ul class="navlist">
+                        <li> <a href="about.html" class="nav_link">About</a></li>
+                        <li> <a href="" class="nav_link">Program</a></li>
+                        <li> <a href="" class="nav_link">Join</a></li>
+                        <li> <a href="" class="nav_link">Sponsor</a></li>
+                        <li> <a href="" class="nav_link">News</a></li>
+                        <li> <a href="" class="nav_link"><button class="home_button">CM Campain</button></a></li>
+                    </ul>
+                </div>  
+            </nav>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please remove white `space` on top of your `header` part and also please `add` correct design to your main `navigation` elements like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+            </time>
+
+            <P class="venue">@National Museum of Uganda, Art Center Nabi and more</P>
+        </section>
+
+
+        <section class="program">
+            <h2 class="prog_heading">
+                Main Program
+            </h2>
```
Comment (mukhammadrizooff) [inline index.html:100]: - [ ] [REQUIRED] Good job so far! Please add `orange line` under to `heading` like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+            </div>
+
+            <div class="see_prog">
+                SEE THE WHOLE PROGRAM
+            </div>
+        </section>
+
+        
+        <section class="speakers" id="speakers">
+            <h3 class="speaker_header">Featured Speakers</h3>
```
Comment (mukhammadrizooff) [inline index.html:176]: - [ ] [REQUIRED] Good job so far! Kindly add correct `design`(color, fonts...) like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+                <div class="speaker_info">
+                  <div class="speaker_name">Radia Perlman</div>
+                  <br>
+                  <div class="speaker_profession">American network engineer</div>
+                  <hr class="speaker_them">
+                  <div class="speaker_detail">Radia Joy Perlman is an American computer programmer and network engineer.</div>
+                </div>
+            </figure>
+                        
+            </div>  -->
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Please remove `Unnecessary` comments 🙏🏼

File: about.html [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+                    <div class="log2">
+                        <img class="log2_img" src="images/logo.png" alt="conference logo">
+                        <div class="log2_title">
+                            <div class="log2_title1">CM</div>
+                            <div class="log2_title2">Summit 2015</div>
+                        </div>
+                    </div>
+
+                    <div class="logo_date">14th-17th Oct | K'la, Uganda</div>
```
Comment (mukhammadrizooff) [inline about.html:140]: - [ ] [REQUIRED] Good job so far! Please fix space `arrangement of elements`  on mobile version like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), I `compare` and `highlight` mistakes below the picture:

image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+            <div class="summit_img">
+                <div class="summit_img_one">
+                    <div class="year"><strong>2013</strong></div> <br>Kampala Programmers Forum 2013 in Buenos Aires
+                </div>
+    
+    
+                <div class="summit_img_two">
+                    <div class="year"><strong>2011</strong></div> <br>Kampala Programmers Forum 2013 in Warsaw
+                </div>
```
Comment (mukhammadrizooff) [inline about.html]: - [ ] [REQUIRED] Good job so far! Please `add` required color fiter to your 2 pictures like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 


image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+                    <div class="small_trade">CM</div>
+                    <div class="trade_two">Coding Minds <br>
+                        International Summit
+                    </div>
+                </div>            
+           
+            <div class="left_info">
+                2015 Coding Minds International Summit
+                <br>Some rights reserved
+            </div>
```
Comment (mukhammadrizooff) [inline about.html:204]: - [ ] [REQUIRED] Good job so far! Kindly `About` page's footer colors different than `Home` page but you use same design both pages therfore please fix it  and check again [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @FrankMugagga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Descriptive `Commit` history ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @FrankMugagga , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**